build-bfb

```diff
@@ -45,8 +44,7 @@ while [ "$1" != -- ]; do
         --uart) shift; uart="$1" ;;
         --sys) shift; sys="$1" ;;
         --root) shift; root=$1 ;;
-        --mmc-cap) shift; mmc_cap=$1 ;;
-        --spi-img) shift; spi_img=$1 ;;
+        --causple) shift; capsule=$1 ;;
```

Comment:
typo "capsule"

Reply:
Done. BTW, Spencer will remove build-bfb from bfscripts in one of the pending pull request.